laravel/composer.json

```diff
@@ -23,7 +23,10 @@
         ],
         "psr-4": {
             "App\\": "app/"
-        }
+        },
+        "files": [
```

Comment:
I don't think it's necessary to declare it as a file here, since it's in the app folder which uses psr-4. :thinking: